doc/modules/preprocessing.rst

```diff
@@ -248,6 +248,36 @@ a :class:`KernelCenterer` can transform the kernel matrix
 so that it contains inner products in the feature space
 defined by :math:`phi` followed by removal of the mean in that space.
 
+.. _preprocessing_transformer:
+
+Non-linear transformation
+=========================
+
+Like scalers, :class:`QuantileTransformer` puts each feature into the same
+range or distribution. However, by performing a rank transformation, it smooths
+out unusual distributions and is less influenced by outliers than scaling
+methods. It does, however, distort correlations and distances within and across
+features.
+
+:class:`QuantileTransformer` and :func:`quantile_transform` provide a
+non-parametric transformation based the quantile function to map the data to a
+uniform distribution::
+
+  >>> from sklearn.datasets import load_iris
+  >>> iris = load_iris()
+  >>> X, y = iris.data, iris.target
+  >>> quantile_transformer = preprocessing.QuantileTransformer()
+  >>> X_trans = quantile_transformer.fit_transform(iris.data)
+
+It is also possible to map the transformed data to a normal distribution by
+setting ``output_distribution='normal'``::
+
+  >>> X_trans = preprocessing.quantile_transform(X, output_distribution='normal')
+
+Thus the median of the input becomes the mean of the output, centered at 0. The
+normal output is clipped so that the input's maximum and minimum do not become
+infinite under the transformation.
```

Comment:
I think this should give more details on the default values of the clipping boundaries.